navigation/ca_collaborative_move_base/config/dwa_local_planner_params.yaml

```diff
@@ -0,0 +1,56 @@
+DWAPlannerROS:
```

Comment:
Since this file doesn't have any modification w.r.t. the original file, we could call the original one instead of the copy.

Reply:
discussed and decided to keep it as it would require further tuning as we continue testing